frontend/package.json

```diff
@@ -26,8 +26,12 @@
     "test-gui-openshift": "yarn run test-suite --suite crud --params.openshift true",
     "test-gui": "yarn run test-suite --suite all",
     "test-suite": "ts-node -O '{\"module\":\"commonjs\"}' ./node_modules/.bin/protractor integration-tests/protractor.conf.ts",
-    "test-cypress": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
-    "test-cypress-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress": "yarn run test-cypress-common && yarn run test-cypress-olm",
+    "test-cypress-headless": "yarn run test-cypress-common-headless && yarn run test-cypress-olm-headless",
+    "test-cypress-common": "cd packages/integration-tests-cypress && cypress open --env openshift=true",
+    "test-cypress-common-headless": "cd packages/integration-tests-cypress && node --max-old-space-size=4096 ../../node_modules/.bin/cypress run --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
+    "test-cypress-olm": "cd packages/operator-lifecycle-manager/integration-tests-cypress && cypress open --config-file cypress-olm.json --env openshift=true",
```

Comment:
Pls see [dev-console cypress PR](https://github.com/openshift/console/pull/6776/files#diff-da6498268e99511d9ba0df3c13e439d10556a812881c9d03955b2ef7c6c1c655R32) changes.

We are using `test-cypress-console...` instead of `..common`, but great minds do think alike :-)